egui_glium/src/lib.rs

```diff
@@ -580,10 +580,10 @@ impl EguiGlium {
         (needs_repaint, shapes)
     }
 
-    pub fn paint(
+    pub fn paint<T: glium::Surface>(
         &mut self,
         display: &glium::Display,
-        target: &mut glium::Frame,
+        target: &mut T,
```

Comment:
Let's replace generics with dynamic dispatch instead to keep down own monomorphization overhead (larger code and slower compilation).

```suggestion
        target: &mut dyn glium::Surface,
```

here and below

Reply:
He @emilk, thanks for reply :) That was also my thought to make it dynamic dispatch, but it's not possible as Surface trait is not object-safe.

Reply:
Oh, ok then :)